From 1a47e66fb850b3bad58dd8f46f5a970557792821 Mon Sep 17 00:00:00 2001
From: sinanmohd <sinan@sinanmohd.com>
Date: Tue, 12 May 2026 22:46:13 +0530
Subject: [PATCH] fix: align toolbar text

---
 src/main.rs | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/main.rs b/src/main.rs
index 2b168d7..3c8a4ee 100644
--- a/src/main.rs
+++ b/src/main.rs
@@ -237,7 +237,7 @@ fn main() {
             .unwrap_or("?");
 
         tooltip += &format!(
-            "{} {} {} {} {} {}%\n",
+            "{} {} {} {} {} {}%\n\n",
             if args.nerd { "󰖜" } else { "🌅" },
             format_ampm_time(day, "sunrise", args.ampm),
             if args.nerd { "󰖛" } else { "🌇" },
@@ -261,7 +261,7 @@ fn main() {
             }
 
             let mut tooltip_line = format!(
-                "{} {} {} {}",
+                "{}  \t{}  \t{}  \t{}",
                 format_time(hour["time"].as_str().unwrap(), args.ampm),
                 if args.nerd {
                     WEATHER_CODES_NERD
-- 
2.53.0

